packages/table/src/table.tsx

```diff
@@ -467,22 +466,22 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     private resizeSensorDetach: () => void;
 
     private refHandlers = {
-        cellContainer: (ref: HTMLElement) => (this.cellContainerElement = ref),
-        columnHeader: (ref: HTMLElement) => (this.columnHeaderElement = ref),
-        mainQuadrant: (ref: HTMLElement) => (this.mainQuadrantElement = ref),
-        quadrantStack: (ref: TableQuadrantStack) => (this.quadrantStackInstance = ref),
-        rootTable: (ref: HTMLElement) => (this.rootTableElement = ref),
-        rowHeader: (ref: HTMLElement) => (this.rowHeaderElement = ref),
-        scrollContainer: (ref: HTMLElement) => (this.scrollContainerElement = ref),
+        cellContainer: (ref: HTMLElement | null) => (this.cellContainerElement = ref),
+        columnHeader: (ref: HTMLElement | null) => (this.columnHeaderElement = ref),
+        mainQuadrant: (ref: HTMLElement | null) => (this.mainQuadrantElement = ref),
+        quadrantStack: (ref: TableQuadrantStack | null) => (this.quadrantStackInstance = ref),
+        rootTable: (ref: HTMLElement | null) => (this.rootTableElement = ref),
+        rowHeader: (ref: HTMLElement | null) => (this.rowHeaderElement = ref),
+        scrollContainer: (ref: HTMLElement | null) => (this.scrollContainerElement = ref),
     };
 
-    private cellContainerElement: HTMLElement;
-    private columnHeaderElement: HTMLElement;
-    private mainQuadrantElement: HTMLElement;
-    private quadrantStackInstance: TableQuadrantStack;
-    private rootTableElement: HTMLElement;
-    private rowHeaderElement: HTMLElement;
-    private scrollContainerElement: HTMLElement;
+    private cellContainerElement: HTMLElement | null;
+    private columnHeaderElement: HTMLElement | null;
+    private mainQuadrantElement: HTMLElement | null;
+    private quadrantStackInstance: TableQuadrantStack | null;
+    private rootTableElement: HTMLElement | null;
+    private rowHeaderElement: HTMLElement | null;
+    private scrollContainerElement: HTMLElement | null;
```

Comment:
we don't use `strictNullChecks` yet so these changes have no effect on the compiler.

Reply:
it doesnt effect anything for the compiler but it is more correct w.r.t. to the react typings. it will also get a head start when bp does switch to strictNullChecks